python/py-pymc3/Portfile

```diff
@@ -36,7 +36,9 @@ if {${name} ne ${subport}} {
                     port:py${python.version}-patsy \
                     port:py${python.version}-scipy \
                     port:py${python.version}-theano-pymc \
-                    port:py${python.version}-typing_extensions
+                    port:py${python.version}-typing_extensions \
+                    port:py${python.version}-semver \
+                    port:py${python.version}-cachetools
```

Comment:
please keep the dependencies alphabetically sorted

Reply:
Thanks, done.